scripts/local/.env

```diff
@@ -0,0 +1,90 @@
+# PCS_IOTHUBREACT_HUB_PARTITIONS=4
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_HUB_NAME
+PCS_IOTHUBREACT_HUB_ENDPOINT
+PCS_IOTHUBREACT_HUB_PARTITIONS
+
+# -----------------------------------------------------------------------------
+# Storage Account information:
+# {Your storage account} > Access keys
+#
+# Example: 
+# PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT=storagetest123
+# PCS_IOTHUBREACT_AZUREBLOB_KEY=12345...6789==
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT
+PCS_IOTHUBREACT_AZUREBLOB_KEY
+
+# -----------------------------------------------------------------------------
+# Storage Endpoint Suffix: 
+# {Your storage account} > Properties > Primary Blob Service Endpoint > value
+# in the URI after the "blob." (eg. `core.windows.net`)
+#
+# Example:
+# PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX=core.windows.net
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX
+
+# -----------------------------------------------------------------------------
+# DocumentDb Conn String:
+# {Your Document DB} > Keys > Primary Connection String
+# Note: use this value for all 3 environment variables below
+#
+# Example:
+# PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING=AccountEndpoint=https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;
+# PCS_TELEMETRY_DOCUMENTDB_CONNSTRING=AccountEndpoint=https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;
+# PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING=AccountEndpoint=https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;
+# -----------------------------------------------------------------------------
+PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING
+PCS_TELEMETRY_DOCUMENTDB_CONNSTRING
+PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING
+
+# -----------------------------------------------------------------------------
+# Auth is disabled for local development
+# -----------------------------------------------------------------------------
+PCS_AUTH_REQUIRED=false
+
+# -----------------------------------------------------------------------------
+# To add a bing map key, please follow the directions here:
+# https://msdn.microsoft.com/library/ff428642.aspx
+# -----------------------------------------------------------------------------
+PCS_BINGMAP_KEY=static
+
+# -----------------------------------------------------------------------------
+# You can allow requests to another domain with the PCS_CORS_WHITELIST, see
+# the following link for more information:
+# https://docs.microsoft.com/aspnet/web-api/overview/security/enabling-cross-origin-requests-in-web-api
+# -----------------------------------------------------------------------------
+PCS_CORS_WHITELIST=
```

Comment:
super nit, why does this one have an = behind it and the rest don't?

Reply:
Not sure. It was like that in the .NET version when I copied it over.